From 020e701d8d581b9b68f97aa3c39995a8fe9082ce Mon Sep 17 00:00:00 2001
From: Masahiro Yamada <yamada.masahiro@socionext.com>
Date: Thu, 26 Oct 2017 21:24:20 +0900
Subject: [PATCH 069/105] mpc85xx: freescale: remove CONFIG_ADDR_STREAMING

This macro is defined, but not referenced by anyone.

I did not touch config_whitelist.txt - the CONFIG will be dropped
by the next re-sync.

Signed-off-by: Masahiro Yamada <yamada.masahiro@socionext.com>
Reviewed-by: York Sun <york.sun@nxp.com>
---
 include/configs/C29XPCIE.h | 1 -
 include/configs/P1010RDB.h | 1 -
 2 files changed, 2 deletions(-)

diff --git a/include/configs/C29XPCIE.h b/include/configs/C29XPCIE.h
index d6982d31a9..7e805ecff0 100644
--- a/include/configs/C29XPCIE.h
+++ b/include/configs/C29XPCIE.h
@@ -109,7 +109,6 @@
 #define CONFIG_L2_CACHE			/* toggle L2 cache */
 #define CONFIG_BTB			/* toggle branch predition */
 
-#define CONFIG_ADDR_STREAMING		/* toggle addr streaming */
 
 #define CONFIG_ENABLE_36BIT_PHYS
 
diff --git a/include/configs/P1010RDB.h b/include/configs/P1010RDB.h
index 13edd0ac9c..adb4a18a70 100644
--- a/include/configs/P1010RDB.h
+++ b/include/configs/P1010RDB.h
@@ -205,7 +205,6 @@
 #define CONFIG_L2_CACHE			/* toggle L2 cache */
 #define CONFIG_BTB			/* toggle branch predition */
 
-#define CONFIG_ADDR_STREAMING		/* toggle addr streaming */
 
 #define CONFIG_ENABLE_36BIT_PHYS
 
-- 
2.13.6

